src/components/Button.jsx

```diff
@@ -0,0 +1,10 @@
+import './../index.css'
+function Button ({label, onClick, disabled}) {
+    return(
+        <button type='button' onClick={onClick} disabled={disabled}>
+            {label}
+        </button>
+    );
+}
+
+export default Button;
```

Comment:
Dit component is nu een doorgeefluik. Het bespaart je (nauwelijks) code door het op deze manier te doen in plaats van direct het button element te gebruiken